pkg/web/templates/transactions.html

```diff
@@ -31,7 +31,7 @@ <h1 class="font-bold text-2xl">Transaction Inbox</h1>
           </button> 
         </div>
       </div>
-      <a href="/v1/transactions/export" download class="btn btn-sm bg-black text-white hover:bg-black/80">Export Transactions</a>
+      <a href="/v1/transactions/export" download class="max-w-fit btn btn-sm bg-black text-white hover:bg-black/80">Export Transactions</a>
```

Comment:
`max-w-fit` ensures that the `Export Transactions` button doesn't appear wider than the one that appears above it.